### PR TITLE
Updated Footer Data

### PR DESCRIPTION
**Pull request by sumitd94**
This is fix for my Previous PR #15

### CODE REVIEW

File: components/footer/footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.infoRepo {
+  font-weight: 100;
+  margin: 0 auto;
+  text-align: center;
+}
```
Comment (invalidtoken) [inline components/footer/footer.module.scss:5]: The styles looks good.

File: components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import classNames from './footer.module.scss';
```
Comment (invalidtoken) [inline components/footer/index.js:2]: Good job importing classNames properly.

File: components/footer/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+const Footer = () => {
+  return (
+    <footer>
+	   <p className={classNames.infoRepo}>The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
+	</footer>
```
Comment (invalidtoken) [inline components/footer/index.js:8]: @sumitd94 this is not formatted properly. Can you please make sure to run `npm run lint`, `npm run format` before pushing your code.

Comment (invalidtoken) [reply components/footer/index.js:8]: Ideally, this is handled by pre-commit hooks but I don't know how this got missed.

**Comment by sumitd94** [comment]
Added a fix for this in PR #22

**Pull request closed without merging**